src/cfnlint/rules/resources/LimitNumber.py

```diff
@@ -35,7 +35,7 @@ def match(self, cfn):
         # Check number of resources against the defined limit
         resources = cfn.get_resources()
         if len(resources) > LIMITS['resources']['number']:
-            message = 'Maximum number of resources ({0}) exceeded'
-            matches.append(RuleMatch(['Resources'], message.format(LIMITS['resources']['number'])))
+            message = 'Maximum number of resources ({0}) exceeded ({1})'
```

Comment:
Does the language seem backwards with the parameters?

Maximum number of resources (# of Resources in the template) exceeded (the Resource LIMIT)?
or how about
The number of resources (len(resources)) exceeded the limit of (resource limit)?